02-peer-protocol.md

```diff
@@ -209,20 +208,19 @@ signature, it will broadcast the funding transaction.
 
 1. type: 34 (`funding_created`)
 2. data:
-    * [8:temporary-channel-id]
-    * [32:txid]
-    * [1:output-index]
+    * [32:funding-txid]
+    * [1:funding-output-index]
     * [64:signature]
 
 #### Requirements
 
-The sender MUST set `temporary-channel-id` the same as the `temporary-channel-id` in the `open_channel` message.  The sender MUST set `txid` to the transaction ID of a non-malleable transaction, which it MUST NOT broadcast, and MUST set `output-index` to the output number of that transaction which corresponds the funding transaction output as defined in [BOLT #3](03-transactions.md#funding-transaction-output), and MUST set `signature` to the valid signature using its `funding-pubkey` for the initial commitment transaction as defined in [BOLT #3](03-transactions.md#commitment-transaction).
+The sender MUST set `funding-txid` to the transaction ID of a non-malleable transaction, which it MUST NOT broadcast, and MUST set `funding-output-index` to the output number of that transaction which corresponds the funding transaction output as defined in [BOLT #3](03-transactions.md#funding-transaction-output), and MUST set `signature` to the valid signature using its `funding-pubkey` for the initial commitment transaction as defined in [BOLT #3](03-transactions.md#commitment-transaction).
 
 The recipient MUST fail the channel if `signature` is incorrect.
 
 #### Rationale
 
-The `output-index` can only be 1 byte, since that's how we'll pack it into the channel-id used throughout the protocol.  The limit of 255 outputs should not be overly burdensome.
+The `funding-output-index` can only be 1 byte, since that's how we'll pack it into the channel-id used throughout the gossip protocol.  The limit of 255 outputs should not be overly burdensome.
```

Comment:
> can only be 1 byte, since that's how we'll pack it into the channel-id used throughout the gossip protocol

We actually currently use `2-bytes` in the gossip protocol for `channel-id`s IIRC. However as you note, I think `1-byte` is sufficient for de-multiplexing channel updates.